### PR TITLE
stress: remove custom loader and use esbuild to build

### LINKED ISSUE #542: Bots - 4th Week of July - Maintenance and Updates #1
### Description

We currently have bots running against gamma

channel: https://app.gamma.towns.com/t/10a38bcf15ab6b94d404c201dee9f67c6428c0ecb10000000000000000000000/channels/20a38bcf15ab6b94d404c201dee9f67c6428c0ecb1a166f49d6787eb5dd4e1b1

action: https://github.com/HereNotThere/harmony/actions/workflows/Run_river_stress_tests.yml

logs: https://app.datadoghq.com/logs?query=env%3Agamma%20service%3Astress-test-node%20&agg_m=count&agg_m_source=base&agg_t=count&cols=env%2Cservice%2Ccontainer_index&fromUser=true&index=%2A&messageDisplay=inline&refresh_mode=sliding&saved-view-id=2705025&storage=hot&stream_sort=desc&viz=stream&from_ts=1722029116522&to_ts=1722030016522&live=true

The stress tests are at a breaking point / no longer working as expected

Expected behaviour
- 40 clients across 2 containers boot up
- first client posts a message to the main channel
- all other clients wait for that message, emoji it, then start chatting in the two chat channels
- after timeout duration they stop and send a summary message back to the thread on the main message in the first channel

First things first - get up to latest way of doing things

Brian has figured out how to run the sdk using esbuild and an es wasm loader
here's the line in workers/notification-service/package.json 
```
"compile": "node esbuild.config.mjs",
```
we should delete my goofy custom loader and use this instead

Running tests locally:
you can run `./packages/stress/scripts/localhost_demo.sh` in river
you can run `./packages/stress/scripts/localhost_setup_chat.sh`and then copy the parameters printed and run `packages/stress/scripts/localhost_chat.sh` 
if you have a private key you can run `packages/stress/scripts/gamma_chat.sh`

for these tasks you only need to worry about the demo

- [ ] remove custom loader, use esbuild

### PR DESCRIPTION
**Pull request by miguel-nascimento**
Closes #542

### CODE REVIEW

File: packages/stress/README.md [hunk 1]
```diff
@@ -12,3 +12,7 @@ node: 119.62s user 17.02s system 779% cpu 17.533 total // 7x speedup!!!
 # Directions
 
 Use files in scripts dir to launch load tests
+
+# Hasks
+
+the wasm generator generates olm.wask with a build artifact in the name, we rename it in the build step so the code can import it. Embedding the wasm either didn't work or didn't make a difference.
```
Comment (brianathere) [inline packages/stress/README.md:18]: sp: olm.wask -> olm.wasm

Did you look at https://esbuild.github.io/api/#asset-names?

File: packages/stress/esbuild.config.mjs [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+build({
+  entryPoints: ["./src/start.ts", "./src/demo.ts"],
+  bundle: true,
+  sourcemap: "inline",
+  platform: "node",
+  target: "node20",
+  format: "cjs",
+  outdir: "dist",
+  outExtension: { ".js": ".cjs" },
+  plugins: [wasmLoader()],
```
Comment (brianathere) [inline packages/stress/esbuild.config.mjs:13]: An alternative to address the olm.wasm being renamed is to pass { mode: "embedded" } to the wasmLoader and the WASM will be inlined into the output js.

File: packages/stress/README.md (hunk 1 above)
Comment (miguel-nascimento) [reply packages/stress/README.md:18]: thank you!!

File: packages/stress/esbuild.config.mjs (hunk 2 above)
Comment (texuf) [reply packages/stress/esbuild.config.mjs:13]: We tried that and it didn’t work.

File: packages/stress/esbuild.config.mjs [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  ignoreAnnotations: true,
+  assetNames: "[name]",
```
Comment (texuf) [inline packages/stress/esbuild.config.mjs:15]: Nice! You fixed it.!

**Review by texuf — APPROVED** [review]

**Pull request merged**